app/MMB/src/navigation/DetailScreen.js

```diff
@@ -151,7 +151,6 @@ export default class DetailsScreen extends React.Component<Props, State> {
    * Do proper navigation transition on mobile (narrow) devices.
    */
   openMenuOnMobile = (key: string) => {
-    // $FlowExpectedError: Flow type for this navigation seems to be quite challenging. I am skipping it on purpose.
```

Comment:
It was challenging because we had checked here for `RouteNames` so we couldn't use a different than existing screen name. This check is also gone now and I am not sure if it's a good idea (you are not getting errors even thou you should - see?). But if so then you should remove the `RouteNames` type as well.

Reply:
Mmm I am not entirely sure why there was a flow error here? We were allowing to navigate to a `routeName` that does not exist? How? Was it working?

Reply:
> We were allowing to navigate to a routeName that does not exist? How?

Just the Flow type didn't exist because it's dynamic here. Otherwise, these pages exist. I don't remember correctly and I was probably just lazy but it works on every other place thanks to the `RouteNames` type.